edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc

```diff
@@ -6,18 +6,19 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can upgrade your managed {sno} cluster with the image-based upgrade through {ztp-first}.
-
-When you deploy the {lcao} on a cluster, an `ImageBasedUpgrade` CR is automatically created.
-You update this CR to specify the image repository of the seed image and to move through the different stages.
+You can use a single resource on the hub cluster, the `ImageBasedGroupUpgrade` custom resource (CR), to manage an imaged-based upgrade on a selected group of managed clusters through all stages.
+{cgu-operator-first} reconciles the `ImageBasedGroupUpgrade` CR and creates the underlying resources to complete the defined stage transitions, either in a manually controlled or a fully automated upgrade flow.
 
 // Lifecycle Agent (LCA)
 
-include::modules/ztp-image-based-upgrade-prep.adoc[leveloffset=+1]
+include::modules/ztp-image-based-upgrade-concept.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
+* xref:../../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-3-expired-certs.adoc#dr-scenario-3-recovering-expired-certs_dr-recovering-expired-certs[Recovering from expired control plane certificates]
+
+////
```

Comment:
For the merge reviewer: These links are commented out temporarily as they're related to the procedure that is updated in a separate PR. I will review if we still need these after rebasing https://github.com/openshift/openshift-docs/pull/80111 with the current PR merged